### ISSUE TITLE
Improve a11y on internal dashboards

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#12615]
https://dash.wellcomecollection.org could really use an a11y audit from our end and benefit from improvements. Basic things such as focus on buttons on keyboard navs are not in place, so I'm sure we could do a lot more.

It could also benefit from UX/UI review.

These dashboards (especially Toggles and Prismic linting) are accessed regularly by colleagues across the Collections and should be made accessible. 

Sub-tickets could be created from this one to add improvements slowly and surely.

**Comment by rcantin-w** [comment]
- Search by id
- Add date of toggle creation
- make "public" clearer

**Comment by rcantin-w** [comment]
> Search by id 

This is already possible

**Comment by rcantin-w** [comment]
All the above is done and deployed
